### ISSUE TITLE
 control in windows gmae presses down by itself

### ISSUE THREAD

**Issue by gutodlima** [batocera-linux/batocera-emulationstation#1794]
### Concerned project

Batocera

### Distribution Version

Batocera v40

### Architecture

Linux

### Issue description

When I choose a Windows game when accessing the game the controller keeps pressing down on its own. This is only happening in Windows games, as this doesn't even happen in the menu on other systems. I use a zero delay USB card.

### Logs and data

https://github.com/user-attachments/assets/072cb6bb-3904-4f3e-99bf-256c960f8380

**Comment by rdmneto** [comment]
Did you got a solution? Im having the same problem here…

**Comment by fabricecaruso** [comment]
If it does not occurs inside EmulationStation, it's not related to EmulationStation, so I can't help, sorry.
It's probably related to batocera-linux with WINE ( windows games )
Test a newer version, and if this persists, open a new issue in the batocera-linux repository.

**Comment by gutodlima** [comment]
> Did you got a solution? Im having the same problem here…

Yes, I deleted the controls configuration file and then configured it again. I believe that for some reason, as I took it from a ready-made image, there is this bug.

**Comment by fabricecaruso** [comment]
perfect
